src/System.Runtime.Serialization.Xml/tests/Configurations.props

```diff
@@ -5,6 +5,7 @@
       netstandard;
       netcoreapp;
       uap;
+      netfx;
```

Comment:
Can you please name the new project `canonicalization.tests.csproj`? 
Yes, the `configuration.props` for the new project would need `netfx`.
Let's not worry about the test for ReflectionOnly mode for now.

Thanks!